overlay_error.py

```diff
@@ -24,7 +20,7 @@ def on_connect(self, client, userdata, flags, rc):
         super().on_connect(client, userdata, flags, rc)
 
         # Forcing an exception to occur
-        with CameraException(self.client, self.backend_name):
+        with self.exception_handler:
```

Comment:
One last thing! (sorry) Let's add exception handling by default for `on_connect` by moving this line to `_on_connect`.

Reply:
I'm sorry, not quite still... By moving the error handling to `overlay.py` we no longer need to do `with self.exception_handler` in this file